src/components/scripts/popupWithConfirmation.js

```diff
@@ -0,0 +1,32 @@
+import Popup from "./Popup";
+
+export default class ConfirmationOnClick extends Popup {
```

Comment:
Please, name `ConfirmationOnClick` according to the check list `PopupWithConfirmation`